.github/workflows/dev-test-run.yml

```
@@ -0,0 +1,66 @@
+name: E2E Test on Dev
+
+description: |
```

actionlint (1.7.4)

unexpected key "description" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)